### DISCUSSION TITLE
关于onebot生态应用案例的补充

### DISCUSSION THREAD

**Discussion by mgsky1** [howmanybots/onebot#139] [应用 / SDK]
这是我写的一个基于`Nonebot`的QQ群机器人，它叫FG，最具特色的功能是使用机器学习算法提取聊天关键词。目前有两个分支，一个是主分支，只有提取关键词功能，另外一个分支是为某科幻群定制的版本，这个版本功能更加丰富，可以欢迎新人，微软小冰（支持文字/图片）智能聊天引擎的接入等～目前正考虑将微软小冰的特性接入主分支。
目前在Github获得17个star，望能补充进`应用案例`模块～
项目主页：https://github.com/mgsky1/FG
主分支：https://github.com/mgsky1/FG/tree/mirai
某科幻群的定制分支：https://github.com/mgsky1/FG/tree/hhx_mirai
效果图：

![](https://raw.githubusercontent.com/mgsky1/FG/mirai/assets/02.png)

**Comment by stdrc** [comment]
直接按原格式提个 pull request 吧

**Comment by mgsky1** [comment]
> 直接按原格式提个 pull request 吧

已提交PR
